src/test/java/itest/GraphQLTest.java

```diff
@@ -224,260 +214,381 @@ void testStartRecordingMutationOnSpecificTarget() throws Exception {
 
         DeleteMutationResponse actual =
-                mapper.readValue(resp.bodyAsString(), DeleteMutationResponse.class);
+                mapper.readValue(resp2.bodyAsString(), DeleteMutationResponse.class);
 
-        MatcherAssert.assertThat(actual.data.targetNodes, Matchers.hasSize(1));
+        MatcherAssert.assertThat(actual.getData().getTargetNodes(), Matchers.hasSize(1));
 
-        TargetNode node = actual.data.targetNodes.get(0);
+        TargetNode node = actual.getData().getTargetNodes().get(0);
 
-        MatcherAssert.assertThat(node.recordings.active.data, Matchers.hasSize(1));
-        MatcherAssert.assertThat(node.recordings.archived.data, Matchers.hasSize(1));
-        MatcherAssert.assertThat(node.recordings.archived.aggregate.count, Matchers.equalTo(1L));
-        MatcherAssert.assertThat(node.recordings.archived.aggregate.size, Matchers.greaterThan(0L));
+        MatcherAssert.assertThat(
+                node.getTarget().getRecordings().getActive().getData(), Matchers.hasSize(1));
+        MatcherAssert.assertThat(
+                node.getTarget().getRecordings().getArchived().getData(), Matchers.hasSize(1));
+        MatcherAssert.assertThat(
+                node.getTarget().getRecordings().getArchived().aggregate.count,
+                Matchers.equalTo(1L));
+        MatcherAssert.assertThat(
+                node.getTarget().getRecordings().getArchived().aggregate.size,
+                Matchers.greaterThan(0L));
 
-        ActiveRecording activeRecording = node.recordings.active.data.get(0);
-        ArchivedRecording archivedRecording = node.recordings.archived.data.get(0);
+        ActiveRecording activeRecording =
+                node.getTarget().getRecordings().getActive().getData().get(0);
+        ArchivedRecording archivedRecording =
+                node.getTarget().getRecordings().getArchived().getData().get(0);
 
-        MatcherAssert.assertThat(activeRecording.name, Matchers.equalTo("graphql-itest"));
-        MatcherAssert.assertThat(activeRecording.doDelete.name, Matchers.equalTo("graphql-itest"));
+        MatcherAssert.assertThat(activeRecording.name, Matchers.equalTo("test"));
+        MatcherAssert.assertThat(activeRecording.doDelete.name, Matchers.equalTo("test"));
 
         MatcherAssert.assertThat(
                 archivedRecording.name,
-                Matchers.matchesRegex(
-                        "^es-andrewazor-demo-Main_graphql-itest_[0-9]{8}T[0-9]{6}Z\\.jfr$"));
+                Matchers.matchesRegex("^selftest_test_[0-9]{8}T[0-9]{6}Z\\.jfr$"));
     }
 
     @Test
     @Order(8)
     void testNodesHaveIds() throws Exception {
+
+        // Create a mock instance of RootNode
+        RootNode rootNode = mock(RootNode.class);
+
+        // Create instances of DiscoveryNode and set properties manually
+        DiscoveryNode node1 = new DiscoveryNode();
+        node1.id =
+                1L; // Setting ID directly if it's public; if not, use setter methods if available
```

Comment:
I'm not sure this comment is really informative, and also the newline seems unnecessary.